### ISSUE TITLE
Future maintainer

### ISSUE THREAD

**Issue by 128keaton** [vegardit/prisma-generator-nestjs-dto#259]
Hello, I'd love to maintain this project seeing as how I use it quite a bit. 

I have a fork located [here](https://github.com/128keaton/prisma-generator-nestjs-dto) that I created to add some options

Funnily enough, I noticed that this would effectively close #92

**Comment by tommyc38** [comment]
@128keaton  I'd be happy to help after December. I am going to be starting a big Nest.js project and have been searching for sort of the holy grail of fullstack development (e.g. one source source of truth for db, dto, swagger, and validation that can be used on both the frontend and backend).  I would really like to bring validation via class-validator into this project as an option in addition to enhancing the current project.  Let me know if you are open to that.

**Comment by 128keaton** [comment]
@tommyc38 That sounds fantastic, I've been wanting something similar to create standardized interfaces for filtering/pagination too, so we have plenty of ideas.

**Comment by ChristopherCapito** [comment]
I suggest linking up with https://github.com/Brakebein/prisma-generator-nestjs-dto

He has some very cool additions with relation to compound types and such. might be worth a conversation. used his fork extensively

**Comment by Arkanii** [comment]
Agree with @ChristopherCapito

I've seen many forks and it can be interesting to focus on a multi-person project, and I think @Brakebein's one is a good candidate in terms of features and popularity.

What do you think about that ?

I mention all people who have done something or wanted to maintain the project :

@Brakebein
@128keaton 
@tommyc38 
@botika
@liliwei25

**Comment by tommyc38** [comment]
I agree.  I think @Brakebein's repo looks promising.  I also didn't realize he already has support for class-validator.  I think it makes sense to partner up there to extend the package further.

**Comment by Brakebein** [comment]
Thanks for the credits. Indeed, my fork had evolved quite a lot, since I use it in some of my projects and added the features I needed. But many others also contributed from time to time with suggestions or pull requests (see changelog for all additions and fixes).

I am open to synchronize with you. Do you have any ideas how to proceed? Or what are your ideas how to contribute?

**Comment by zzau13** [comment]
https://github.com/botika/prisma-generator-nestjs-dto Here the class-validators work, I use babel, but the original structure doesn't work for me and I have to do a refactor. 
It is in development but I am using it in production, it has errors due to the original structure that I will fix in a couple of weeks.

**Comment by Arkanii** [comment]
Is there a way to merge your two project's ideas together and maintain the new one / existing one as a team ?

**Comment by okonon** [comment]
I'd be interested to contribute with my fork as well forked from @Brakebein . I had a need to add `ApiHideProperty` to hide createdBy and updatedBy fields from swagger but not fail validation with `whitelist` and `forbidNonWhitelisted` set to true.

**Comment by zzau13** [comment]
I am willing to give the little code that I have done to a team and take it together. Now, I am applying solid principles on the structure for easy extension.

**Comment by Brakebein** [comment]
> I'd be interested to contribute with my fork as well forked from @Brakebein . I had a need to add `ApiHideProperty` to hide createdBy and updatedBy fields from swagger but not fail validation with `whitelist` and `forbidNonWhitelisted` set to true.

@okonon I added `ApiHideProperty` last month based on your request (https://github.com/Brakebein/prisma-generator-nestjs-dto/issues/23).

**Comment by okonon** [comment]
@Brakebein super! i really appreciate it your implementation is way cleaner then mine
